### ISSUE TITLE
ReferenceError: PluginPostStatusInfo is not defined

### ISSUE THREAD

**Issue by ivanlopez** [10up/distributor#185]
After updating Gutenberg to Version 3.6.2 when I try and edit an existing post that was created with a previous version of Gutenberg (3.4.0) I'm getting the "The editor has encountered an unexpected error." Once I disable the Distributor I'm able to edit the page.

`ReferenceError: PluginPostStatusInfo is not defined
    at u (https://test.com/wp-content/plugins/distributor/dist/js/gutenberg-status-plugin.min.js?ver=1.2.3:1:675)
    at yh (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:95:430)
    at lg (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:120:88)
    at mg (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:120:386)
    at gc (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:127:202)
    at vb (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:230)
    at ub (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:65)
    at zd (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:124:449)
    at ra (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:123:319)
    at Object.enqueueSetState (https://test.com/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:189:231)`

![screen shot 2018-08-20 at 2 21 47 pm](https://user-images.githubusercontent.com/1428016/44358936-6ad85180-a484-11e8-94b7-78ac8a8c31b6.png)

**Comment by dkotter** [comment]
@ivanlopez I've made a PR to fix this, #186. If you have a second to test that out and verify it fixes your issue, that would be great.

I know for me, I was using Gutenberg v3.4.0 and didn't upgrade. I pulled in a post from a site that wasn't using Gutenberg and when trying to edit that post, I received this same error. So it seems like it isn't necessarily related to upgrading to the latest version, as it impacts at least 3.4.0 (I didn't test older versions).
